draft-ietf-quic-transport.md

```diff
@@ -1722,6 +1722,10 @@ it can associate the peer's response with the corresponding PATH_CHALLENGE.
 
 On receiving a PATH_CHALLENGE frame, an endpoint MUST respond immediately by
 echoing the data contained in the PATH_CHALLENGE frame in a PATH_RESPONSE frame.
+An endpoint MUST NOT store or retransmit a PATH_RESPONSE frame, as a new
+PATH_CHALLENGE frame will be sent if another PATH_RESPONSE frame is needed.  An
+endpoint MAY choose to limit the number of pending PATH_RESPONSE frames
```

Comment:
I think that this amounts to ignoring PATH_CHALLENGE. It might be better to say that instead.

Reply:
It'd actually be slightly better to respond to the later ones, if possible.  That being said, if you're in this state, it's likely a bug in the peer or an attack.

Reply:
Clarified in 52957ec. I didn't want to get into which one to respond to on a given CID, since I don't think it makes a huge difference and it makes it a bit more confusing.